ci-operator/templates/cluster-launch-e2e-openshift-jenkins.yaml

```diff
@@ -214,12 +215,14 @@ objects:
         mkdir -p /tmp/output
         cd /tmp/output
 
+        # we are not setting up the telemetry client like in cluster-launch-e2e.yaml since it was
+        # marked as temporary there
+
         # TODO: the test binary should really be a more structured command - most of these flags should be
         #       autodetected from the running cluster.
-        # TODO: bump nodes up to 40 again
         function run-tests() {
           if [[ -n "${TEST_FOCUS:-}" ]]; then
-            ginkgo -v -noColor -nodes="${TEST_PARALLELISM:-30}" $( which extended.test ) -- \
+            ginkgo -v -noColor -nodes="${TEST_PARALLELISM:-3}" $( which extended.test ) -- \
```

Comment:
If you're parameterizing this from the jobs, keep the default as the same value as the other templates (which makes copy and paste easier).

Reply:
we are / will do